Frameworks/AddressBook/ABMultiValueInternal.mm

```diff
@@ -159,4 +159,71 @@ - (CFIndex)getIndexForIdentifier:(ABMultiValueIdentifier)identifier {
     return index;
 }
 
+- (_ABMultiValue*)createMutableCopy {
```

Comment:
>createMutableCopy [](start = 18, length = 17)

Why use this and not conform to `NSMutableCopying` by implementing `mutableCopyWithZone:`